streamparse/dsl/topology.py

```diff
@@ -194,6 +194,8 @@ def _spec_to_flux_dict(spec):
             shell_object = spec.component_object.shell
             flux_dict['constructorArgs'].append([shell_object.execution_command,
                                                  shell_object.script])
+            if not spec.outputs or len(spec.outputs) == 0:
```

Comment:
Super minor nitpick, but could you please get rid of the `or len(spec.outputs) == 0` from this line? It's unnecessary, because if `spec.outputs` is empty, `not spec.outputs` will be true.